packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -55,11 +55,12 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
 
         if (label && labelPlacement && (labelPlacements.includes(labelPlacement))) {
             return html`
-                <span
+                <label
```

Comment:
Part of me wonders if we're building a lot of label functionality into the toggle when it's due to be pulled out as a separate component before we tackle any of the other inputs.

Would it make sense to make the tickets for the label now (after you've merged this PR) rather than wait, as it may mean a bit of consideration in terms of how you structure this component pulling it in?

Reply:
Happy to create some tickets for this. Will speak to you and Raouf after this one is merged 👍